src/Microsoft.VisualStudio.Editor.Razor/DefaultRazorIndentationFactsService.cs

```diff
@@ -105,113 +78,20 @@ internal class DefaultRazorIndentationFactsService : RazorIndentationFactsServic
                     }
                 }
 
-                owningChild = owningParent;
-            }
-
-            return desiredIndentation;
-        }
-
-        private Span LocateOwner(Block root, SourceChange change)
```

Comment:
We do this better on our SYntaxTreeNodes this is basically just inlined duplicate code 🥂